install/_lib.sh

```diff
@@ -6,7 +6,7 @@ log_file="sentry_install_log-`date +'%Y-%m-%d_%H-%M-%S'`.txt"
 exec &> >(tee -a "$log_file")
 
 # Work from /install/ for install.sh, project root otherwise
-if [[ "$(basename $0)" = "install.sh" || "$(basename $0)" = "test.sh" ]]; then
+if [[ "$(basename $0)" = "install.sh"  ]]; then
```

Comment:
Was this necessary to get this working, or a sidecar?

Reply:
`test.sh` doesn't exist anymore so we should not refer to it?

Reply:
Ah, right. And we removed that way earlier on this PR. 😁